src/MCPClient/osdeps/Ubuntu-14.json

```diff
@@ -0,0 +1,49 @@
+{
+  "_comments": [
+    "Add any comments here",
+    "Include here keys, repos, and packages required by this component not already in archivematicaCommon osdeps file"
+  ],
+  "osdeps_repokeys": [
+    { "id": "0x5236CA08",
+      "url": "https://packages.archivematica.org/GPG-KEY-archivematica",
+      "validate_certs": "no"
+    }
+  ],
+  "osdeps_repos": [
+    "deb [arch=amd64] http://packages.archivematica.org/1.6.x/ubuntu-externals trusty main",
+    "deb http://archive.ubuntu.com/ubuntu/ trusty multiverse",
+    "deb http://archive.ubuntu.com/ubuntu/ trusty-security universe",
+    "deb http://archive.ubuntu.com/ubuntu/ trusty-updates multiverse"
+  ],
+  "osdeps_packages": [
+  { "name": "atool", "state": "latest"},
+  { "name": "bagit", "state": "latest"},
+  { "name": "bulk-extractor", "state": "latest"},
+  { "name": "ffmpeg", "state": "latest"},
```

Comment:
This a difficult question. The current format is based on what the ansible apt / yum modules takes as input (which in turn are based on the way apt and yum OS commands work), it is possible to specify a fixed version or to install the "latest" version available, but not possible to specify a "minimum" version.